bulbs/recirc/static/recirc/js/series-recirc.js

```diff
@@ -0,0 +1,36 @@
+var VideoRecircList = function() {
+  $videoItem = 'video-item';
```

Comment:
Should use `var` to define these variables and also the `$` for variable names is typically used to denote it's a jQuery object, which only one of these is (`$videoRecircList`).  I wouldn't use the `$` prefix for the others.